fragments/comments.js

```diff
@@ -89,3 +90,23 @@ export const COMMENTS = gql`
       }
     }
   }`
+
+export const ITEM_WITH_COMMENTS = gql`
+  ${COMMENT_FIELDS}
+
+  fragment ItemWithComments on Item {
+      ...CommentFields
+      comments {
+          ...CommentFields
+      }
+  }`
+
+export const NEW_COMMENTS = gql`
+  ${ITEM_WITH_COMMENTS}
+
+  query NewComments($rootId: ID!, $after: Date!) {
+    newComments(rootId: $rootId, after: $after) {
+      ...ItemWithComments
```

Comment:
Ideally we'd query `comments { ...CommentsRecursive }` here and fetch the entire new subtree(s) if there is one.

It might be too big of a lift though (up to you), because it'd likely require something like `item_comments_zaprank_with_me` as in https://github.com/stackernews/stacker.news/blob/fb1281dfd2e68bdd63e90225d664b098f4bccf0f/prisma/migrations/20231220004234_dont_like_sats/migration.sql but it would return an array of subtrees rather than just a single (sub)tree.

For *extra* stuff like this you can opt-out of doing it if you want. I just wanted to mention where this is likely headed.